### DISCUSSION TITLE
Cannot use sctp flow in iperf3

### DISCUSSION THREAD

**Discussion by kcn21** [esnet/iperf#1356] [General]
Hello, I want to use SCTP flow for network testing, but in spite of iperf3's 3.1 and newer versions of iperf3 having SCTP support, the server and client command is not executing giving the error of iperf3: unrecognized option '--sctp'. 

In iperf3 module, which I have downloaded from the official website has iperf_sctp.c and iperf_sctp.h files but sctp functionality is still not available. Please help me with this issue.

**Comment by swlars** [comment]
Hi,

Could you tell us a little more about your test? For example, what platform are you using and are you building from source?

It might not be detecting sctp in the operating system. If you're using Linux, you might need to install a few packages to make it visible. 

Thanks!

**Comment by Kaauw** [comment]
a `few packages` mean nothing. But if someone is questionning himself which lib to install on debian based system : [libsctp-dev](https://manpages.debian.org/testing/libsctp-dev/index.html)
